WireAuthentication/Sources/WireAuthenticationUI/Views/Root/Router.swift

```diff
@@ -27,4 +27,25 @@ public protocol Router {
 
     func navigate<Destination: Hashable>(to destination: Destination)
 
+    func present(modal: ModalDestination)
```

Comment:
question: what benefit is there adding this to the router rather than presenting the modal locally from a view? It seems like it achieves the same thing but with more steps.